### PR TITLE
Debug studio

### PR DESCRIPTION
**Pull request by speudusa**
studio added to tools chapter

### CODE REVIEW

File: src/chapters/tools-in-intelliJ/studio.rst [hunk 1]
```diff
@@ -1,4 +1,59 @@
-Studio
-======
+Studio: If It Ain't Broke, Add a Breakpoint!
+============================================
+ 
+The purpose of this studio is to expand your current debugging strategies.  
+First, as a group, you will discuss your current debugging strategies.  
+Next, you will explore the debugger tools addressed in this chapter.
+
+This studio will use a previous studio assignment to explore IntelliJ's Debugger.
```
Comment (gildedgardenia) [inline src/chapters/tools-in-intelliJ/studio.rst:8]: I would provide a direct link to the repository they need to be looking at.

File: src/chapters/tools-in-intelliJ/studio.rst [hunk 2]
```diff
@@ -1,4 +1,59 @@
...
+On your machine, open up your copy of ``lsn7interfaces`` in IntelliJ.
+
+
+
+Part One: Pre-Debugger Debugging
+--------------------------------
+
+Review with the group one error you encountered when working on your version of lesson 7’s studio.
```
Comment (gildedgardenia) [inline src/chapters/tools-in-intelliJ/studio.rst]: lsn7interfaces is a package in the exercises. But this makes it sound like it is part of the lesson 7 studio.

**Review by gildedgardenia — APPROVED** [review]
Looks great!

File: src/chapters/tools-in-intelliJ/studio.rst (hunk 2 above)
Comment (speudusa) [reply src/chapters/tools-in-intelliJ/studio.rst]: yes, it should be the studio.

File: src/chapters/tools-in-intelliJ/studio.rst (hunk 1 above)
Comment (speudusa) [reply src/chapters/tools-in-intelliJ/studio.rst:8]: good idea! thanks!

**Pull request merged**